src/components/NavSection.vue

```diff
@@ -13,15 +13,18 @@
       <li class="nav_search" v-if="showNavSearch ==='true'">
         <form class="hero_search-form"
               role="search"
-              method="post"
+              method="get"
```

Comment:
Maybe I'm missing something, but why are changes to the markup necessary? All of the required elements are present on the page and this should mostly be behavioral changes.

Reply:
Everything looks pretty good (and works well) to me, but I'll defer to @brenoferreira, especially with the changes RE the method and action attributes.